dbfit-java/core/src/main/java/dbfit/api/DbCommand.java

```diff
@@ -0,0 +1,25 @@
+package dbfit.api;
+
+import java.sql.SQLException;
+
+/**
+ * An object representing an executable database operation
+ * with no result returned.
```

Comment:
Even in #463 I think we're using `DbCommand` for functions too. And also it's always possible to have output parameters of non-function stored procedures.

Maybe with that comment my intent was to make some distinction with `DbQuery` and to highlight that results are not retrieved through the interface itself.